internal/services/datafactory/data_factory_linked_service_sftp_resource.go

```diff
@@ -103,31 +121,42 @@ func resourceDataFactoryLinkedServiceSFTP() *pluginsdk.Resource {
 				},
 			},
 
-			"skip_host_key_validation": {
-				Type:     pluginsdk.TypeBool,
-				Optional: true,
+			"passphrase": {
+				Type:          pluginsdk.TypeString,
+				Optional:      true,
+				Description:   "Specify the pass phrase or password to decrypt the private key if the key file or the key content is protected by a pass phrase",
+				ConflictsWith: []string{"password"},
 			},
 
-			"host_key_fingerprint": {
-				Type:         pluginsdk.TypeString,
-				Optional:     true,
-				ValidateFunc: validation.StringIsNotEmpty,
+			"password": {
+				Type:          pluginsdk.TypeString,
+				Optional:      true,
+				Sensitive:     true,
+				ValidateFunc:  validation.StringIsNotEmpty,
+				ConflictsWith: []string{"private_key_content", "private_key_path", "passphrase"},
+				AtLeastOneOf:  []string{"private_key_content", "private_key_path"},
 			},
 
-			"annotations": {
-				Type:     pluginsdk.TypeList,
-				Optional: true,
-				Elem: &pluginsdk.Schema{
-					Type: pluginsdk.TypeString,
-				},
+			"private_key_content": {
+				Type:          pluginsdk.TypeString,
+				Optional:      true,
+				Description:   "Base64 encoded SSH private key content or Azure Key Vault Secret URL. SSH private key should be in OpenSSH format",
+				ValidateFunc:  validate.SSHPrivateKey,
+				ConflictsWith: []string{"private_key_path", "password"},
+				AtLeastOneOf:  []string{"password", "private_key_content", "private_key_path"},
```

Comment:
I think this should also be marked sensitive
```suggestion
			"private_key_content": {
				Type:          pluginsdk.TypeString,
				Optional:      true,
				Sensitive:     true,
				Description:   "Base64 encoded SSH private key content or Azure Key Vault Secret URL. SSH private key should be in OpenSSH format",
				ValidateFunc:  validate.SSHPrivateKey,
				ConflictsWith: []string{"private_key_path", "password"},
				AtLeastOneOf:  []string{"password", "private_key_content", "private_key_path"},
```

Reply:
you're right, it should be, ty